### PR TITLE
fix(KYC): IOS-1187 show correct verification status for new wallets

### PR DESCRIPTION
**Pull request by achtenhagen**
## Objective

Fix the default identity verification status for newly created wallets.

## Description

Previously, the case for a new user was handled in the same way as a user with a `failed` state.

## How to Test

Create a new wallet and launch the settings view. Observe that the label inside the identity verification cell reads `Unverified` as opposed to `Failed`.

## Screenshot/Design assessment

simulator screen shot - iphone x - 2018-09-04 at 16 04 25

## Merge Checklist

- [x] The PR uses a title supported by [.changelogrc](https://github.com/blockchain/My-Wallet-V3-iOS/blob/dev/.changelogrc#L6...L69).
- [x] Areas of technical debt are marked with a `// TICKET:` comment that includes a ticket number.
- [x] All unit tests pass.
- [ ] You have added unit tests.
- [x] New files added are within the correct directory. (e.g. if a file is required for unit tests to compile, be sure it is added to the tests target.)

### CODE REVIEW

File: Blockchain/View Controllers/Settings/SettingsCell.swift [hunk 1]
```diff
@@ -89,18 +89,14 @@ import UIKit
 
         self.detailTextLabel?.frame.origin.x = screenWidth - detailWidth! - 40
 
-        
         self.detailTextLabel?.font = UIFont(name: Constants.FontNames.montserratSemiBold, size: Constants.FontSizes.Tiny)
 
-        
 //        self.detailTextLabel?.layer.cornerRadius = 4
```
Comment (bchrisarriola) [inline Blockchain/View Controllers/Settings/SettingsCell.swift]: Delete?

File: Blockchain/View Controllers/Settings/SettingsViewController.swift [hunk 2]
```diff
@@ -79,7 +77,6 @@ MobileNumberDelegate, WalletAccountInfoDelegate {
         super.viewDidAppear(animated)
         tableView.reloadData()
         self.walletManager.accountInfoDelegate = self
-        preparedIdentityStatus == false
```
Comment (bchrisarriola) [inline Blockchain/View Controllers/Settings/SettingsViewController.swift]: 🤔

Comment (achtenhagen) [reply Blockchain/View Controllers/Settings/SettingsViewController.swift]: What are your thoughts 💭?

Comment (bchrisarriola) [reply Blockchain/View Controllers/Settings/SettingsViewController.swift]: Just strange that this was even checked in (this line wasn't doing anything useful and the property wasn't actually used).

Comment (achtenhagen) [reply Blockchain/View Controllers/Settings/SettingsViewController.swift]: Ah yes, completely unnecessary 🕵️‍♂️

**Pull request merged**